config/manager/deployment.yaml

```diff
@@ -67,6 +67,8 @@ spec:
         imagePullPolicy: IfNotPresent
         name: manager
         resources:
+          limits:
+            memory: 500Mi
```

Comment:
This was explicitly taken out in a recent commit, we don't want a memory limit anymore. Are we doing anything with kubebuilder that regenerates this file? Afaik it's just an initial one time generation, after which we have full control.

Reply:
Hmm, I wonder if we properly remove the memory limit, was it missed in 01_deployment.yaml? It should be gone in both it sounds like.